src/server/svr_func.c

```diff
@@ -656,11 +656,16 @@ cred_name_okay(attribute *pattr, void *pobj, int actmode)
 int
 poke_scheduler(attribute *pattr, void *pobj, int actmode)
 {
-	if (actmode == ATR_ACTION_ALTER) {
-		if (pattr->at_val.at_long)
-			set_scheduler_flag(SCH_SCHEDULE_CMD);
+	if ( pobj == &server || pobj == dflt_scheduler) {
```

Comment:
can this function be moved to sched_func.c? Also as part of this attribute being set we need to confirm that all other mandatory attributes in the scheduler object are correctly set. This includes checking for port number, priv/log directory etc. All this is part of interface 1 of the design proposal.